Makefile

```diff
@@ -192,11 +192,11 @@ install-targets-schedtask: $(BUILD_DIR_SCHEDTASK)/$(SCHEDTASK_INSTALL)
 uninstall-targets-schedtask: $(BUILD_DIR_SCHEDTASK)/$(SCHEDTASK_UNINSTALL)
 	test $(CUR_OS) != Windows || ./$<
 
-# Copies sources to build directory & replace "$INSTALL_PREFIX".
+# Copies sources to build directory & replace "{{ INSTALL_PREFIX }}".
 $(BUILD_DIR)/% : %
 	@${MKDIR_PARENTS} $@
 	cp $< $@
-	sed -i.bak -e 's|$$INSTALL_PREFIX|$(INSTALL_PREFIX)|g' $@; rm $@.bak
+	sed -i.bak -e 's|{{ INSTALL_PREFIX }}|$(INSTALL_PREFIX)|g' $@; rm $@.bak
```

Comment:
This was the important piece of the replace, removing the extra `$` on the sed expression as was previously necessary to escape the dollar.